recipes/crypten/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "crypten" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f9d76bc0ba1468cb77261b8d5486ee875266363310353a18aba948429aa24fda
+  - url: https://raw.githubusercontent.com/facebookresearch/CrypTen/main/requirements.txt
+    sha256: 14569c02e2bb5233d9d75a13646a7e9eaee0a1bcc8e7ffe4044ad9a3511d8eca
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - setuptools
+    - pytest-runner
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+    - pytest-runner
+  run:
+    - future
+    - setuptools
+    - omegaconf >=2.0.6
+    - onnx >=1.7.0
+    - pandas >=1.2.2
+    - python >=3.6
+    - pyyaml >=5.3.1
+    - scipy >=1.6.0
+    - scikit-learn
+    - tensorboard
+    - pytorch >=1.7.0
+    - torchvision >=0.9.1
+    - pytest-runner
+
+test:
+  imports:
+    - crypten
+  requires:
```

Comment:
Typically people require `pip` so that they may run `pip check`. Does your application require pip to be imported? Very likely not. If that is the case, please remove the `requires` section in the test and remove `pip` too!

Reply:
Removed.